tests/csapi/encryption_test.go

```diff
@@ -0,0 +1,95 @@
+package csapi_tests
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"maunium.net/go/mautrix"
+	"maunium.net/go/mautrix/event"
+	"maunium.net/go/mautrix/id"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+)
+
+func TestEncryption(t *testing.T) {
+	deployment := Deploy(t, b.BlueprintOneToOneRoom)
+	defer deployment.Destroy(t)
+
+	alice := deployment.Client(t, "hs1", "@alice:hs1")
+	bob := deployment.Client(t, "hs1", "@bob:hs1")
+
+	encAlice := client.MustCreateEncryptedClient(t, alice)
+	encBob := client.MustCreateEncryptedClient(t, bob)
+
+	// Create a new room and invite Bob
+	roomResp, err := encAlice.Client.CreateRoom(&mautrix.ReqCreateRoom{
```

Comment:
Please try to use the Complement version where possible, we don't want to have to maintain 2 `CreateRoom` flavours in tests.